lib/interface_quda.cpp

```diff
@@ -695,7 +696,7 @@ void initQuda(int dev)
 static cudaGaugeField* createExtendedGauge(cudaGaugeField &in, const int *R, TimeProfile &profile,
 					   bool redundant_comms=false, QudaReconstructType recon=QUDA_RECONSTRUCT_INVALID)
 {
-  profile.TPSTART(QUDA_PROFILE_INIT);
+  // profile.TPSTART(QUDA_PROFILE_INIT);
```

Comment:
Why comment these out?  Was this fixing an issue with timers somewhere?

Reply:
Will check with some through testing.